resources/analysisTools/copyNumberEstimationWorkflow/impute2.sh

```diff
@@ -48,7 +48,7 @@ then
          
          if [[ "$?" != 0 ]]
          then
-         	echo "Non zero exit status for mpileup in impute2.sh"
+         	echo "Non zero exit status for mpileup in impute2.sh" >> /dev/stderr
```

Comment:
Ok, so due to LSF, do we need to echo everything to /dev/stderr?
Also it might be useful to have a method like:
function checkLastExitCode "$messageOnFail" "$throwExitCode"

somewhere in the bash lib

Or to have it more versatile:

function validateValueOrFail "$?|any other value" "$messageOnFail" "$exitCodeOnFail"

Reply:
Your second suggestion would not work. It is to general. (What value?) You could do it with test expressions, though (but only using eval :-( )

Reply:
You'd need to put in an integer of some sort. Otherwise it won't work. But in the usages in this file, it would be possible to replace at least 4 copies.

Reply:
We should put it into an issue. I am not going to change that now, because it is out of scope and @Turtletrok's code anyway.